src/components/MoneyRequestHeader.js

```diff
@@ -117,8 +93,7 @@ function MoneyRequestHeader(props) {
                     personalDetails={props.personalDetails}
                     shouldShowBackButton={props.isSmallScreenWidth}
                     onBackButtonPress={() => Navigation.goBack(ROUTES.HOME, false, true)}
-                    statusBar={isScanning ? getStatusBar() : null}
-                    shouldShowBorderBottom
+                    statusBar={isScanning ? <MoneyRequestHeaderStatusBar /> : null}
```

Comment:
We still don't need a `statusBar` prop. It's better to not create a prop that may never get used. `MoneyRequestHeaderStatusBar` belongs to `MoneyRequestHeader` more than `HeaderWithBackButton`.

```js
<HeaderWithBackButton />
<MoneyRequestHeaderStatusBar />
```
Looks better